src/Pangraph/GraphML/Parser.hs

```diff
@@ -11,10 +11,10 @@ import qualified Pangraph.Internal.XMLTemplate       as PT
 
 -- * Parsing
 
--- | Returns `Pangraph` if it can be parsed from a raw GraphML file.
+-- | Returns 'Pangraph' if it can be parsed from a raw GraphML file.
 parse :: BS.ByteString -> Maybe P.Pangraph
 parse file = either (const Nothing) (PT.hexmlToPangraph PT.graphMLTemplate) (H.parse file)
 
--- | Like `parse` except it throws an error on Nothing.
+-- | Like 'parse' except it throws an error on Nothing.
```

Comment:
Let's make `throws an error on Nothing` a bit more helpful by saying when it actually throws the error (when parse failed).